pkg/subvolume/subvolume.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subvolume
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	exec "github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+)
+
+func List(ctx context.Context, clientsets *k8sutil.Clientsets, clusterNamespace, operatorNamespace string) {
+
+	getpv := `kubectl get pv -o jsonpath={.items[*].spec.csi.volumeAttributes.subvolumeName}`
```

Comment:
```suggestion
	getpv := `kubectl get pv -o jsonpath={.items[*].spec.csi.volumeAttributes.subvolumeName}`
```

Please use kubernetes api to list the pv avoid using bash command as much as possible

Reply:
We don't want to list all the pvs and other details, we just want the subvolumename and I think this would be the fatser way to do it rather then getting the list of pv and then adding filter to it. Let me know what are your thoughts.

Reply:
@yati1998 I what I found in my local testing is the bash command is taking longer time
```
getpv := `kubectl get pv -o jsonpath={.items[*].spec.csi.volumeAttributes.subvolumeName}`
```

you can do something like this which in comparison if faster than bash
```
pvList, err := clientsets.Kube.CoreV1().PersistentVolumes().List(context.Background(), v1.ListOptions{})
	if err != nil {
		fmt.Printf("Error fetching PVs: %v\n", err)
		os.Exit(1)
	}

	for _, pv := range pvList.Items {
		subvolumeName := pv.Spec.CSI.VolumeAttributes["subvolumeName"]
		fmt.Printf("PV Name: %s, SubvolumeName: %s\n", pv.Name, subvolumeName)
	}```

Reply:
thanks